src/domains/printit-app/99_variables.tf

```diff
@@ -184,7 +184,15 @@ variable "app_service_pdf_engine_sku_name_java" {
   default     = "S1"
 }
 
-variable "pdf_engine_app_ha_enabled" {
-  type        = bool
-  description = "(Required) enables the deployment of pdf engine in HA mode"
+### Features flags
+
+variable "is_feature_enabled" {
+  type = object({
+    pdf_engine    = bool
+    pdf_engine_ha = bool
+  })
+  default = {
+    pdf_engine    = false
+    pdf_engine_ha = false
```

Comment:
@jacopocarlini can we define only one pdf_engine service plan block, and use the variables to distinguish the deployment type?

Reply:
ok